src/sentry/tasks/store.py

```diff
@@ -189,6 +189,23 @@ def preprocess_event_from_reprocessing(
     )
 
 
+def increment_low_priority_metrics_counter(cluster, project_id, timestamp, ttl):
+    # Increment the event counter for the given project_id in the given cluster.
+    #
+    # The key is computed from the project_id and the timestamp of the symbolication request, rounded
+    # down to the nearest 10 seconds. If the key is not currently set to expire, it will be set to expire
+    # in ttl seconds.
+    with sentry_sdk.start_span(
+        op="tasks.store.symbolicate_event.low_priority.metrics.counter"
+    ) as span:
+        # Round the time to the nearest 10s
+        timestamp = int(timestamp)
+        timestamp -= timestamp % 10
+        key = f"symbolicate_event_low_priority:{project_id}:{timestamp}"
+        cluster.incr(key)
+        cluster.expire(key, ttl, NX=True)
```

Comment:
Should we use a pipeline for this?  iirc currently this will be two roundtrips to the redis server and will be halved with pipelines.